test/utils.dart

```diff
@@ -25,17 +25,89 @@ const formattedOutput = 'void main() => print("hello");';
 final _indentPattern = RegExp(r'\(indent (\d+)\)');
 final _fixPattern = RegExp(r'\(fix ([a-x-]+)\)');
 
-/// Runs the command line formatter, passing it [args].
-Future<TestProcess> runFormatter([List<String> args]) {
+/// If tool/command_shell.dart has been compiled to a snapshot, this is the path
+/// to it.
+String _commandExecutablePath;
+
+/// If bin/format.dart has been compiled to a snapshot, this is the path to it.
+String _formatterExecutablePath;
+
+/// Compiles format.dart to a native executable for tests to use.
```

Comment:
I would mention the use of `setUpAll` and `tearDownAll` here. The name makes me think it would happen within the call.

Reply:
Done.